.xo-config.js

```diff
@@ -50,7 +43,14 @@ module.exports = {
 				svg: 'always'
 			}
 		],
-		'unicorn/no-fn-reference-in-iterator': 'off'
+
+		// Temporarily disabled
+		'@typescript-eslint/no-invalid-void-type': 'off',
+		'@typescript-eslint/no-extra-non-null-assertion': 'off',
+		'@typescript-eslint/no-unnecessary-condition': 'off',
```

Comment:
This has a lot of false-positives. Mostly because array access doesn't result in `string | undefined`, but rather `string`. I guess I'll have to disable it in XO again...

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/1798

Reply:
Yeah that rule is problematic especially for libraries since it's true that TS users can't pass the wrong type, but regular JS users can. So this might still sense:

```js
function foo(bar: string) {
	if (typeof bar !== 'string') { // no-unnecessary-condition!! … or is it?
		throw new TypeError('Not a duck')
	}
}
```